nginx/rpc.conf

```diff
@@ -0,0 +1,17 @@
+server {
+  listen 8545;
+
+  # stamp request start time (milliseconds since epoch)
+  set $req_start_ms $msec;
+  proxy_set_header X-Req-Start-Ms $req_start_ms;
```

Comment:
As discussed on the call, this approach of adding a timestamp to the header makes sense, but we also need to do something with the heading inside `op-move`. Maybe building off the metrics PR we could expose a "real" request duration metric which is based on the header timestamp. Then a difference between the `op-move` measured duration and this "real" duration would be a measure of congestion on the tokio side.

We also discussed the possibility of using this header timestamp to perform exact local replays of devnet runs, but that can be left as future work.

Reply:
Added the new metrics in fae2936 and created #543 for further work.